whylog/config/__init__.py

```diff
@@ -28,6 +35,39 @@ def _filter_rule_set(self, parsers_list):
     def _get_locations_for_logs(self, logs_types_list):
         pass
 
+    def _load_parsers(self):
+        parsers_definitions = self._load_file_with_config(self._parsers_path)
+        log_type_manager = LogTypeManager()
+        return [
+            self._create_parser_object(parser_definition, log_type_manager)
+            for parser_definition in parsers_definitions
+        ]
+
+    def _load_file_with_config(self, path):
+        with open(path, "r") as config_file:
+            return list(yaml.load_all(config_file))
+
+    def _create_parser_object(self, parser_definition, log_type_manager):
```

Comment:
In all other functions "_object" is skipped, so skip it here too, for consistency.